cmd/publishing-bot/publisher.go

```diff
@@ -155,6 +155,19 @@ func (p *PublisherMunger) construct() error {
 			}
 			return strings.Join(depStrings, ",")
 		}
+		if len(repoRules.PublishScript) == 0 {
+			// TODO: Move this to main.go?
```

Comment:
yes, please

Reply:
actually can't be done because this depends on the rules to be loaded.